main-site/components/sponsors-section/SponsorsSection.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import SecondaryButton from '../../../shared-ui/components/secondary-button/SecondaryButton';
+import {
+  StyledSponsorsSectionContainer,
+  StyledDiamondLogo,
+  StyledGoldLogos,
+  StyledSilverLogos,
+  StyledSponsorsHeader,
+  StyledContactText,
+  StyledTreasureChest
+} from './SponsorsSection.styles';
+import DiamondLogoDesktop from '../../images/diamond-level-logo-desktop.png';
+import GoldLogosDesktop from '../../images/gold-level-logos-desktop.png';
+import SilverLogosDesktop from '../../images/silver-level-logos-desktop.png';
+import TreasureChest from '../../images/treasure-chest.svg';
+import PrimaryButton from '../../../shared-ui/components/primary-button/PrimaryButton';
+
+const SponsorsSection: React.FC = () => {
+  return (
+    <StyledSponsorsSectionContainer>
+      <StyledTreasureChest src={TreasureChest} alt="treasure chest" />
+      <StyledSponsorsHeader>2023 Sponsors</StyledSponsorsHeader>
+      <div>
+        <SecondaryButton btnText="Diamond Level" isClickable={false} />
+        <br />
+        <StyledDiamondLogo src={DiamondLogoDesktop} alt="diamond level logo" />
+      </div>
+      <div>
+        <SecondaryButton btnText="Gold Level" isClickable={false} />
+        <br />
+        <StyledGoldLogos src={GoldLogosDesktop} alt="gold level logos" />
+      </div>
+      <div>
+        <SecondaryButton btnText="Silver Level" isClickable={false} />
+        <br />
+        <StyledSilverLogos src={SilverLogosDesktop} alt="silver level logos" />
+      </div>
+      <StyledContactText>
+        Interested in joining the HackBeanpot universe? <br /> <br /> Reach out
```

Comment:
this seems like the space theme wording from last year? do we want to keep it

Reply:
nice catch! I'll change it to desert exploration